src/materials/nodes/PointsNodeMaterial.js

```diff
@@ -12,31 +22,30 @@ class PointsNodeMaterial extends NodeMaterial {
 
 	}
 
+	/**
+	 * Constructs a new points node material.
+	 *
+	 * @param {Object?} parameters - The configuration parameter.
+	 */
 	constructor( parameters ) {
 
 		super();
 
+		/**
+		 * This flag can be used for type testing.
+		 *
+		 * @type {Boolean}
+		 * @readonly
+		 * @default true
+		 */
 		this.isPointsNodeMaterial = true;
 
-		this.lights = false;
-		this.transparent = true;
```

Comment:
@sunag You have added this line in #23600 although points are by default opaque according to `PointsMaterial`. Do you recall why you have added this?

Reply:
I don't remember, but removing it seems correct in the current context for me.